### PR TITLE
[VSEE-998] App-scanning 1.6 docs

### PR DESCRIPTION
**Pull request by rxl7906**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
1.6

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: scst-scan/ivs-custom-samples.hbs.md
Comment (chaomonica) [inline scst-scan/ivs-custom-samples.hbs.md:3]: Hi docs team, please let us know if improvements need to be made to this line. Thank you!

File: scst-scan/app-scanning-observing.hbs.md
Comment (ryan-a-baker) [inline scst-scan/app-scanning-observing.hbs.md:3]:
```suggestion
This topic guides you through observing  Supply Chain Security Tools - Scan 2.0 as it it progresses through a scan.  This will help a user understand the each step of the scanning process.
```

File: scst-scan/getting-started.hbs.md
Comment (ryan-a-baker) [inline scst-scan/getting-started.hbs.md:5]:
```suggestion
[Installing App-Scanning](./install-app-scanning.hbs.md)
```

File: scst-scan/getting-started.hbs.md
Comment (ryan-a-baker) [inline scst-scan/getting-started.hbs.md:7]:
```suggestion
[Integrating your own scanner](./integrate-app-scanning.hbs.md)
```
Making these more outcome focused.

File: scst-scan/install-app-scanning.hbs.md
Comment (ryan-a-baker) [inline scst-scan/install-app-scanning.hbs.md:3]:
```suggestion
The following sections describe how to install SCST - Scan 2.0.  Since SCST-Scan 2.0 is in beta, it is not currently installed with any profiles and must be installed following these directions.  
```

File: scst-scan/install-app-scanning.hbs.md
Comment (ryan-a-baker) [inline scst-scan/install-app-scanning.hbs.md:10]:
```suggestion
- Install the [Tekton component](../tekton/install-tekton.hbs.md). Tekton will already be installed if you installed TAP via a profile based installation in both the Full and Build Profiles.
```

File: scst-scan/integrate-app-scanning.hbs.md
Comment (ryan-a-baker) [inline scst-scan/integrate-app-scanning.hbs.md:3]:
```suggestion
The default configuration for out of the box test + scan supply chain uses Supply Chain Security Tools - Scan 1.0. This topic describes how to integrate Supply Chain Security Tools - Scan 2.0 with the out of the box test + scan supply chain, as well as bring your own scanner by creating an ImageVulnerabilityScan template.
```

File: scst-scan/integrate-app-scanning.hbs.md
Comment (ryan-a-baker) [inline scst-scan/integrate-app-scanning.hbs.md:7]:
```suggestion
Before you can integrate Supply Chain Security Tools - Scan 2.0 with the out of the box supply chain:
```

File: scst-scan/integrate-app-scanning.hbs.md
Comment (ryan-a-baker) [inline scst-scan/integrate-app-scanning.hbs.md:8]:
```suggestion
- Select an ImageVulnerabilityScan. You can either bring your own scanner by creating a custom ImageVulnerabilityScan template or select from the provided samples:
```

I think selecting an option is more appropriate than create right?  Since if you are using one of the sample templates you don't need to create it?

File: scst-scan/integrate-app-scanning.hbs.md
Comment (ryan-a-baker) [reply scst-scan/integrate-app-scanning.hbs.md:9]: I'm trying to emphasize "bring your own scanner" and use consistent verbiage throughout the docs.

File: scst-scan/integrate-app-scanning.hbs.md
Comment (ryan-a-baker) [inline scst-scan/integrate-app-scanning.hbs.md:11]:
```suggestion
- [Create a ClusterImageTemplate](./clusterimagetemplates.hbs.md). Incorporated the ImageVulnerabilityScan template into a ClusterImageTemplate.
```

File: scst-scan/integrate-app-scanning.hbs.md
Comment (ryan-a-baker) [inline scst-scan/integrate-app-scanning.hbs.md:22]:
```suggestion
    Where `CLUSTERIMAGETEMPLATE` is the name of the ClusterImageTemplate with the embedded ImageVulnerabilityScan using the scanner of your choice.
```

File: scst-scan/integrate-app-scanning.hbs.md
Comment (ryan-a-baker) [inline scst-scan/integrate-app-scanning.hbs.md:96]:
```suggestion
  - Specifying the `--image` flag configures the workload to skip the source scanning step if has been enabled by using the specified pre-built image. Currently SCST - Scan 2.0 only performs image scanning. There are specific requirements for pre-built images. For more details see [Configure your workload to use a prebuilt image](../scc/pre-built-image.hbs.md)
```
On the fence about removing this entirely, but I think the last part if important to capture.

File: scst-scan/ivs-create-your-own.hbs.md
Comment (ryan-a-baker) [inline scst-scan/ivs-create-your-own.hbs.md:1]:
```suggestion
# Bring your own scanner using an ImageVulnerabilityScan template
```

File: scst-scan/ivs-custom-samples.hbs.md
Comment (ryan-a-baker) [inline scst-scan/ivs-custom-samples.hbs.md:15]:
```suggestion
For information about bringing your own scanner, see [Integrate your own scanner](./app-scanning-alpha.hbs.md#integrate-your-own-scanner).
```

File: scst-scan/integrate-app-scanning.hbs.md
Comment (ryan-a-baker) [reply scst-scan/integrate-app-scanning.hbs.md:1]: Would it be better as enabling app-scanning?  I'm trying to be consistent in the use of "bring your own scanner" but this isn't really that, this is really just the outline of how to enable it over Scan 1.0.

My thoughts right now are to merge it is as it is to get some of the important information out there, then iterate over it.  There are such substantial updates in this that I want to get it in and not block it by names of sections.

File: scst-scan/getting-started.hbs.md
Comment (chaomonica) [reply scst-scan/getting-started.hbs.md]: Changed to [Installing SCST - Scan 2.0]

File: scst-scan/integrate-app-scanning.hbs.md
Comment (chaomonica) [reply scst-scan/integrate-app-scanning.hbs.md]: Updated to be more specific on the supply chain being selected.

**Comment by rxl7906** [comment]
@a-peek4 Hi Alison, we have a lengthy PR to reorganize our scan 2.0 beta docs after feedback. Could you please review it when you get a chance and let us know if you have any concerns/questions?

**Comment by ryan-a-baker** [comment]
Thank you for making all the changes @chaomonica!  This PR looks good to me, and as discussed, I'll iterate some more on the overview once this is all merged.

Is there anything else holding up this PR?  There are a few things that are removed in this PR that we want in for people who are testing as it is causing some confusion.

**Review by ryan-a-baker — APPROVED** [review]

**Pull request merged**
